tendrl/notifier/notification/__init__.py

```diff
@@ -122,9 +123,21 @@ def run(self):
                 for alert in alerts:
                     alert.tags = json.loads(alert.tags)
                     if alert.delivered == "False":
+                        lock = etcd.Lock(
+                            NS._int.client,
```

Comment:
use NS._int.wclient  for locking. How is locking the etcd read client going to help?

Reply:
i will change it

Reply:
correct, my mistake i will change this